dapps/react-dapp-v2/src/contexts/ClientContext.tsx

```diff
@@ -155,7 +155,7 @@ export function ClientContextProvider({
           // Create a flat array of all requested chains across namespaces.
           const standaloneChains = Object.values(requiredNamespaces)
             .map((namespace) => namespace.chains)
-            .flat() as string[];
+            .flat();
```

Comment:
Looks like removing this broke the build:
https://vercel.com/walletconnect/react-dapp-v2/4uuvDkZGLdCdv9rcbS4hqWcfJ3aK

```suggestion
            .flat() as string[];
```